src/pages/blog/[...slug].astro

```diff
@@ -15,14 +16,38 @@ const { Content } = await entry.render();
 ---
 
 <Layout title={entry.data.title}>
-  <article class="prose dark:prose-invert max-w-none">
-    <h1 class="text-4xl font-extrabold tracking-tight lg:text-5xl">{entry.data.title}</h1>
-    <p class="text-lg text-gray-500">
-      Published on:{" "}
-      <time datetime={entry.data.pubDate.toISOString()}>
-        {entry.data.pubDate.toLocaleDateString("en-US", { year: 'numeric', month: 'long', day: 'numeric' })}
-      </time>
-    </p>
-    <Content />
+  <article class="w-full">
+    <!-- Header Section -->
```

Comment:
Another pointless comment. Just refactor into components if we want to split these parts up